cfg.sample.toml

```diff
@@ -29,6 +29,9 @@ name = ""
 # who has r+ rights?
 reviewers = ["barosl", "graydon"]
 
+# who has 'try' rights? (try, retry, force, clean, prioritization)
+try = []
```

Comment:
I don't like _reviewers or _maintainers, since that's precisely the power they don't have, "reviewing" :smile: 

Any alternate suggestion?